### ISSUE TITLE
node:internal/modules/cjs/loader:1154   return process.dlopen(module, path.toNamespacedPath(filename));

### ISSUE THREAD

**Issue by cantiir2** [Automattic/node-canvas#1871]
how to fix this module with problem this :(
Error: The specified module could not be found.
\\?\C:\Users\Mochamad Kadarusman\Downloads\Compressed\Sociality Bot\Sociality Bot\node_modules\canvas\build\Release\canvas.node
    at Object.Module._extensions..node (node:internal/modules/cjs/loader:1154:18)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:816:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at Object.<anonymous> (C:\Users\Mochamad Kadarusman\Downloads\Compressed\Sociality Bot\Sociality Bot\node_modules\canvas\lib\bindings.js:3:18)
    at Module._compile (node:internal/modules/cjs/loader:1095:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1124:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:816
![Capture](https://user-images.githubusercontent.com/36468037/127445769-636f00dd-df2c-478f-bf5b-0cb64a00ccda.PNG)

**Comment by rhasan33** [comment]
+1

**Comment by LinusU** [comment]
`npm rebuild` maybe?

**Comment by Grifed-source** [comment]
For me it's doesn't work too with a rebuild

**Comment by LinusU** [comment]
@Grifed-source do you get any output from your rebuild command? If you nuke `node_modules` and run `npm install`, do you get any output/errors?

**Comment by Grifed-source** [comment]
npm install

**Comment by Grifed-source** [comment]
It deleted a lot of modules, otherwise there were no errors.

**Comment by zbjornson** [comment]
Please post all of the info requested in the issue template (Node version, OS and OS version, Canvas version). This error can happen for a variety of reasons and it's not possible to troubleshoot without more info.

**Comment by Shirishx01** [comment]
I got this error when i defined canvacord in my rank code i asked about this in the support server they said this error coming from node-canvas so can anyone please help me with this i'm stuck!! 
![image](https://user-images.githubusercontent.com/81070901/129510034-b1ee06c9-7b88-4ea5-863f-f7024a0ba2bd.png)

**Comment by A3PvdM** [comment]
Anyone?

**Comment by Pickles-wave** [comment]
I'm running into this issue as well，i got a solution.

If you're an x86 program, this issue may occur, and if you look at canvas.node with [Dependency Walker](http://www.dependencywalker.com/), you'll see that it may be missing **freetype6.dll** and **intl.dll**, in fact, this comes from the GTK, but when you compile the 32-bit canvas, the library name it copies is libfreetype-6.dll and libintl-8.dll, which is the name of the 64-bit compilation, and the copy fails here, but it is just a warning, and if you look at the full log of npm install, you will see this warning. 

To put it simply, copying all the .dlls in C:\GTK\bin into node_modules\canvas\build\Release seems to be a workaround.
